sympy/integrals/tests/test_deltafunctions.py

```diff
@@ -35,8 +35,9 @@ def test_deltaintegrate():
     assert deltaintegrate(DiracDelta(x)**2, x) == DiracDelta(0)*Heaviside(x)
     assert deltaintegrate(y*DiracDelta(x)**2, x) == \
         y*DiracDelta(0)*Heaviside(x)
-    assert deltaintegrate(DiracDelta(x, 1)**2, x) is None
-    assert deltaintegrate(y*DiracDelta(x, 1)**2, x) is None
+    assert deltaintegrate(DiracDelta(x, 1)**2, x) == -DiracDelta(0, 2)*Heaviside(x)
```

Comment:
I hope I didn't merge prematurely here. I thought things were addressed with the latest commit.